samples/apps/basic/custom_endpoints/registry.h

```diff
@@ -0,0 +1,599 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+
+// CCF
+#include "ccf/app_interface.h"
+#include "ccf/common_auth_policies.h"
+#include "ccf/ds/hash.h"
+#include "ccf/http_query.h"
+#include "ccf/json_handler.h"
+#include "ccf/version.h"
+
+#include <charconv>
+#define FMT_HEADER_ONLY
+#include <fmt/format.h>
+
+// Custom Endpoints
+#include "ccf/bundle.h"
+#include "ccf/endpoint.h"
+#include "ccf/endpoints/authentication/js.h"
+#include "ccf/js/core/context.h"
+#include "ccf/js/core/wrapped_property_enum.h"
+#include "ccf/js/extensions/ccf/consensus.h"
+#include "ccf/js/extensions/ccf/converters.h"
+#include "ccf/js/extensions/ccf/crypto.h"
+#include "ccf/js/extensions/ccf/historical.h"
+#include "ccf/js/extensions/ccf/host.h"
+#include "ccf/js/extensions/ccf/kv.h"
+#include "ccf/js/extensions/ccf/request.h"
+#include "ccf/js/extensions/ccf/rpc.h"
+#include "ccf/js/extensions/console.h"
+#include "ccf/js/extensions/math/random.h"
+#include "ccf/js/modules.h"
+#include "ccf/service/tables/modules.h"
+#include "endpoint.h"
```

Comment:
```suggestion
#include "apps/basic/custom_endpoints/endpoint.h"
```

Think it's worth being explicit with include paths, as we have several `endpoint.h`s now.

Reply:
Actually that does not work, because we -I include and src, but not samples. I'll make it a comment. I think this eventually disappears anyway once we've set the public API.